### PR TITLE
Jk/cumulus 2318 feature rds

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2318: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2318)

feature-rds version of https://github.com/nasa/cumulus/pull/2023

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: tf-modules/cumulus/variables.tf [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Required
 
+variable "async_operation_image" {
+  description = "docker image to use for Cumulus async operations tasks"
+  type = string
+  default = "cumuluss/async-operation:27"
```
Comment (flamingbear) [inline tf-modules/cumulus/variables.tf:6]: Should this get updated to 32 now also?  I see they're over written in the example variables.tf var.  Wasn't sure though.

Comment (flamingbear) [reply tf-modules/cumulus/variables.tf:6]: I assume it's getting overwritten by the declaration in example/cumulus-tf/variables.tf, but should they match?

**Review by flamingbear — CHANGES_REQUESTED** [review]
Looks the same as #2023 
Same question about Secret of the ASYNC
And should the defaults for async-operation container version match in the two terraform locations?

File: tf-modules/cumulus/variables.tf (hunk 1 above)
Comment (Jkovarik) [reply tf-modules/cumulus/variables.tf:6]: Yeah, they should, I'll update that.

Comment (Jkovarik) [reply tf-modules/cumulus/variables.tf:6]: Actually, this already *was* updated: https://github.com/nasa/cumulus/pull/2024/commits/7d1972c24bcd9e6580183c11a0a5992052c8b093

Comment (Jkovarik) [reply tf-modules/cumulus/variables.tf]: Fixed, thanks for the feedback.   6fac2303e

**Review by flamingbear — APPROVED** [review]
lgtm

**Pull request merged**